src/schema/nodes/section.ts

```diff
@@ -58,7 +58,8 @@ const choosePageBreakStyle = (element: HTMLElement): number => {
 
 export const section: NodeSpec = {
   // NOTE: the schema needs paragraphs to be the default type, so they must explicitly come first
-  content: 'section_label? section_title (paragraph | element)* section*',
+  content:
+    'section_label? section_title (paragraph | element)* (graphical_abstract_section | section | footnotes_section | bibliography_section)*',
```

Comment:
Instead of listing the different section types, can we use a group instead? if all these nodes have a group "sections", then we should be able to use "sections*".

Reply:
Done